<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Approvisionnement</class>
 <widget class="QWidget" name="Approvisionnement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion d'approvisionnement</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/GestionStock.png</normaloff>:/images/GestionStock.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Style pour les QPushButton */
QPushButton {
    background-color: none;  /* Vert */
    color:  #4CAF50;
    border: 2px solid #4CAF50;
    border-radius: 10px;
    padding: 10px;
}

QPushButton:hover {
    background-color:white;  /* Vert plus foncé */
	   font-weight: bold;
}

QPushButton:pressed {
    background-color: #388E3C;  /* Vert encore plus foncé */
}

/* Style pour les QLineEdit */
QLineEdit {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour les QDateEdit */
QDateEdit {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour les QSpinBox */
QSpinBox {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour les QDoubleSpinBox */
QDoubleSpinBox {
    background-color: white;
    border: 1px solid #4CAF50;
    padding: 5px;
    border-radius: 5px;
    color: black;
}

/* Style pour le QTableView */
QTableView {
    background-color: white;
    gridline-color: #4CAF50;
    border: 1px solid #4CAF50;
	color:black;

}
QTableView:pressed {
	bacground-color:rgb(188, 188, 188);
	 gridline-color: #4CAF50;
    border: 1px solid #4CAF50;
	color:black;
}

QHeaderView::section {
    background-color: #4CAF50;
    color: white;
    padding: 4px;
    border: 1px solid #4CAF50;
}

/* Style pour les QLabel */
QLabel {
    color: black;
    font-weight: bold;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color:#4CAF50;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;PharmaStock&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Code Produit </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="code_lineEdit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Nom du produit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nom_lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Type du produit</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Quantité</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="qte_spinBox"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Date d'expiration</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateExpiration_dateEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Date d'approvisionnement</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDateEdit" name="dateApprovisionnement_dateEdit"/>
     </item>
     <item row="6" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="ajouter_pushButton">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Ajouter au panier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="annuler_pushButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color:none;
    color:;
}</string>
         </property>
         <property name="text">
          <string>Annuler </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="supprimer_pushButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color:none;
    color:red;
}</string>
         </property>
         <property name="text">
          <string>Supprimer du panier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="vider_pushButton">
         <property name="text">
          <string>Vider le panier</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="typeProduit_lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="enregistrer_pushButton_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Enregistrer un approvisionnement</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprime_pushButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Imprimer  la liste des approvisionnements</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imprimeContenu_pushButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Imprimer  le contenu d' approvisionnement</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Gestion d'approvisionnement&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QTableView" name="tableView_2"/>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">background-color:#4CAF50;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>474</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="4">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Nos approvisionnements&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_pushButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Rechercher un approvisionnement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rechercher_lineEdit"/>
     <widget class="QPushButton" name="retour_pushButton">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Contenu du panier&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deconnexion_pushButton">
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>Déconnexion</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>474</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
